starfish/munge.py

```diff
@@ -24,6 +24,12 @@ def max_proj(stack):
     return im
```

Comment:
This is a helper class used to convert spot_attributes (dataframe) into a multi-index that is usable with the IntensityTable